modules/cactus-web/src/MenuBar/MenuBar.tsx

```diff
@@ -80,18 +85,21 @@ const variantMap: VariantMap = {
 
     > ${ScrollButton}, [role='menubar'] > li > [role='menuitem'] {
       color: ${color('white')};
-      border: ${border('base')};
-      border-bottom: ${border('base', { thin: '3px', thick: '4px' })};
+      border: ${border('transparent', { thin: '1px', thick: '2px' })};
 
       &:hover:not([aria-disabled]),
       &[aria-expanded='true'] {
+        ${insetBorder('white', 'bottom', { thin: '2px', thick: '2px' })};
         border-color: ${color('white')};
       }
-
       &:focus {
-        border: ${border('white')};
+        border-color: ${color('white')};
       }
     }
+
+    [role='menubar'] > li > [role='menuitem'] {
+      padding: calc(24px - ${borderSize()}) calc(16px - ${borderSize()}) 25px;
```

Comment:
This is fine, but I think it might be slightly less confusing if you did it like this (matter of opinion?):
```suggestion
      padding: ${borderSize({ thin: '23px 15px 25px', thick: '22px 14px 25px' })};
```

Reply:
No you're right. I also want to get rid of calc() because [IE has trouble with math](https://caniuse.com/?search=calc). I think this particular instance is fine, but I still like to avoid it.